### PR TITLE
Removed explicit mode for multi-lora

### PR DESCRIPTION
**Pull request by oandreeva-nv**
explicit model load was causing failures in tests with vllm==v0.5.0.post1

### CODE REVIEW

File: ci/L0_multi_gpu/multi_lora/multi_lora_test.py [hunk 1]
```diff
@@ -119,7 +119,6 @@ def _test_vllm_model(
         self.triton_client.stop_stream()
 
     def test_multi_lora_requests(self):
-        self.triton_client.load_model(self.vllm_model_name)
```
Comment (Tabrizian) [inline ci/L0_multi_gpu/multi_lora/multi_lora_test.py:122]: Do we know why it doesn't work with explicit mode?

Comment (oandreeva-nv) [reply ci/L0_multi_gpu/multi_lora/multi_lora_test.py:122]: Due to the way vllm evaluates free GPU memory. I'm not sure what exactly changed, but in explicit mode `initialize` fails at:
```
 Error in memory profiling. This happens when the GPU memory was not properly cleaned up before initializing the vLLM instance.
 ```
Unfortunately, I don't have the capacity to investigate what exactly changed in the profiling

Comment (tanmayv25) [reply ci/L0_multi_gpu/multi_lora/multi_lora_test.py:122]: @oandreeva-nv  Have you created ticket for further investigation?

Comment (oandreeva-nv) [reply ci/L0_multi_gpu/multi_lora/multi_lora_test.py:122]: DLIS-7012

**Review by tanmayv25 — APPROVED** [review]

**Pull request merged**